cli/tsc/dts/lib.deno.unstable.d.ts

```diff
@@ -2050,10 +2050,10 @@ declare namespace Deno {
 
     /**
      * Close the database connection. This will prevent any further operations
-     * from being performed on the database, but will wait for any in-flight
-     * operations to complete before closing the underlying database connection.
+     * from being performed on the database, and interrupt any in-flight
+     * operations immediately.
      */
-    close(): Promise<void>;
+    close(): void;
   }
```

Comment:
The updated JSDoc describes a significant difference in behavior, and I don't see any associated runtime changes. Was the description simply incorrect before?

Reply:
@jsejcksn yes, somehow it slipped through the cracks during the review.